package-lock.json

```diff
@@ -12054,7 +12054,7 @@
     },
     "onetime": {
       "version": "1.1.0",
-      "resolved": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
```

Comment:
@adamabeshouse so i think best to revert this change and rebase on https://github.com/cBioPortal/cbioportal-frontend/commit/0b6823c795e659a4fd5745e9b34d6b17ba1683a4

Reply:
yep just waiting for the e2e tests to run so i can see if any failures